src/__tests__/FlashList.test.tsx

```diff
@@ -770,4 +770,33 @@ describe("FlashList", () => {
     flashList.setProps({ disableAutoLayout: true });
     expect(flashList.find(AutoLayoutView)?.props.disableAutoLayout).toBe(true);
   });
+  it("Computed correct scrollTo offset if view position/offset are specified", () => {
```

Comment:
Let's move it to two separate test cases? It will be a little easier to comprehend